sample/build.gradle.kts

```diff
@@ -27,6 +29,22 @@ android {
   testOptions {
     execution = "ANDROIDX_TEST_ORCHESTRATOR"
   }
+  kotlinOptions {
+    jvmTarget = "1.8"
+    useIR = true
+  }
+  buildFeatures {
+    compose = true
+  }
+  composeOptions {
+    kotlinCompilerExtensionVersion = rootProject.extra["compose_version"] as String
+    kotlinCompilerVersion = "1.5.21"
```

Comment:
This is not needed anymore because Compose uses the kotlin compiler defined in the buildscript

Reply:
Removed compose from sample